ignite3/src/main/java/site/ycsb/db/ignite3/AbstractSqlClient.java

```diff
@@ -66,6 +83,18 @@ public void init() throws DBException {
       insertPreparedStatementString = String.format("INSERT INTO %s (%s) VALUES (%s)",
           tableNamePrefix, columnsString, valuesString);
 
+      updateOneFieldPreparedStatementStrings = new HashMap<>();
+      for (String field : valueFields) {
+        updateOneFieldPreparedStatementStrings.put(
+            field,
+            String.format("UPDATE %s SET %s = ? WHERE %s = ?", tableNamePrefix, field, PRIMARY_COLUMN_NAME)
+        );
+      }
+
+      String updateFields = String.join(" = ?, ", valueFields) + " = ?";
```

Comment:
Probably, this would be more readable (I was confused for some time by the current approach: "why do we add an extra `= ?` here?"):
```suggestion
      String updateFields = valueFields.stream()
          .map(v -> String.format("%s = ?", v))
          .collect(Collectors.joining(", "));
```

In the proposed approach it's made more visible that we have N `= ?` strings for N fields, and only N-1 commas.